packages/back-end/src/models/ExperimentModel.ts

```diff
@@ -358,14 +358,21 @@ export async function getSampleExperiment(
 }
 
 export async function createExperiment({
-  data,
+  data: rawData,
   context,
 }: {
   data: Partial<ExperimentInterface>;
   context: ReqContext | ApiReqContext;
 }): Promise<ExperimentInterface> {
+  const data: Partial<ExperimentInterface> = {
+    ...rawData,
+    name: rawData.name?.trim(),
+  };
```

Comment:
Let's get rid of all of the trim stuff from the back end. The front end can do the trim before submitting to the backend.

Reply:
How about API requests? Having the back-end double down on data cleanup and validation is always a good idea as last resource.